.github/actions/deploy/tls_manager.py

```
@@ -0,0 +1,745 @@
+              {{
+                "op": "add",
+                "path": "/spec/template/spec/containers/0/volumeMounts/-",
+                "value": {{
+                  "name": "ca-certs",
+                  "mountPath": "/etc/mysql/ca",
+                  "readOnly": true
+                }}
+              }}
+            ]'''
+
+            print(f'🔧 Patching MariaDB deployment to enable TLS...')
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)